src/kixi/datastore/transport_specs.clj

```diff
@@ -8,11 +8,11 @@
 
 (s/def ::filemetadata-transport
   (s/keys :req-un [::schema-id
-                   ::ms/name
```

Comment:
This means that when the keywords are used in the transport maps, they don't have the namespaces. :req-un is a bit odd, it de-namespaces the keys and checks them. A clue to what's going on is the ::schema-id, rather than ::ss/id like everywhere else.